nymms/schemas/__init__.py

```diff
@@ -105,12 +117,43 @@ def increment_attempt(self):
 
 class Result(StateModel, OriginModel):
     id = StringType(required=True)
-    timestamp = TimestampType(default=time.time)
+    timestamp = TimestampType(default=arrow.get)
     output = StringType()
     task_context = JSONType()
 
```

Comment:
I had to add the following, to avoid the context from being submitted to sdb (too large):

class Options:
        roles = {'sdb': blacklist('task_context')}

where the result is saved, you also need to call to_primative(role='sdb')